webpack.requirejs_jquery_q.config.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
```

Comment:
Let's add a comment here and in the other config file describing what's going on (i.e. other one is minified + includes license comments, etc; this one is not minified, depends on jquery/q being defined in the env, etc)